src/style/themes/base/base-theme.config.js

```diff
@@ -35,7 +35,8 @@ export default (palette) => {
     disabled: {
       border: palette.slateTint(80),
       disabled: blackWithOpacity(0.55),
-      input: palette.slateTint(95)
+      input: palette.slateTint(95),
+      text: blackWithOpacity(0.3)
```

Comment:
Do you know what the `disabled.disabled` colour is used for?

Reply:
I see this is used in `day-picker` files